nlp_profiler/high_level_features/grammar_quality_check.py

```diff
@@ -26,11 +25,11 @@ def apply_grammar_check(heading: str,
 ### Grammar check: this is a very slow process
 ### take a lot of time per text it analysis
 def grammar_check_score(text: str) -> int:
-    if (not isinstance(text, str)) or (len(text.strip()) == 0):
+    if not isinstance(text, str) or not text.strip():
```

Comment:
All of this is good small change, I would keep the parenthese around conditions

The line could have also been written like this:

```python
if not (isinstance(text, str) or text.strip()):
```

See if all the tests pass with the change

Reply:
You haven't applied this? Is sourcery able to parse the above line?

Reply:
I have applied it shows on my end and have committed the same as well.

Reply:
Push it and it will appear here - you may have applied to another branch on #71 but it hasn't reflected on this on